filewatcher/filewatcher.go

```diff
@@ -68,9 +67,7 @@ func (r *Result) watcherLoop(
 			return
 
 		case err := <-watcher.Errors:
-			if logger != nil {
-				logger("watcher error: " + err.Error())
-			}
+			log.Errorw("watcher error", "err", err)
```

Comment:
do we have a preferable way to log errors? does it seems fine?

Reply:
This is fine (as an example). But we do not want to remove the `Wrapper` interface.

There are two parts of the log package:

1. A logger to be used by baseplate.go users (microservices), to log their stuff
2. A way to log some of the library code (other baseplate.go packages), which should be kept to an absolute minimal, like in this case that it's a background goroutine that there's no other way the library can communicate this error to the caller.

Reply:
I returned the `Wrapper` interface back. Thanks!